sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -47,18 +48,16 @@
                     context="{'res_partner_search_mode': 'customer', 'default_is_company': True, 'default_customer_rank': 1}"
                     options='{"always_reload": True}'
                     required="1"
-                    attrs="{'readonly': [('order_partner_id', '!=', False), ('order_id', '!=', False)]}"
+                    readonly="'order_id', 'order_partner_id'"
```

Comment:
You have to do a python statement equivalent to the former attr readonly domain.